travisci/python-linter_harness.sh

```diff
@@ -15,7 +15,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-PYTHON_SOURCE_LOCATIONS=('scripts')
+PYTHON_SOURCE_LOCATIONS=('scripts' 'src/python/lib/ensembl/compara' 'src/python/tests')
```

Comment:
Why not merge both into `src/python` :smile: ?

Reply:
To do what we actually want to do, this script needs some updates (see https://github.com/PyCQA/pylint/issues/352).